hardhat.config.js

```diff
@@ -1,19 +1,24 @@
+require("@nomiclabs/hardhat-etherscan");
 require("@nomiclabs/hardhat-waffle");
+require("hardhat-contract-sizer");
```

Comment:
Try removing these to only leave what we need.
May only need the hardhat-waffle
Move the vyper to package.json